### ISSUE TITLE
Removed the party feature?

### ISSUE THREAD

**Issue by zero5enpai** [pwall2222/NOWT#2]
any specific reason why the showing party feature got removed? 
The one which shows which ppl are in the same squad. Thats like mainly why I used that tracker above others....

**Comment by CreepsoOff** [comment]
It didn't get removed, still working on my side.

**Comment by pwall2222** [comment]
I have yet to fix this issue because it's something related to decoding the presences so it's gonna be tough. Don't expect this to work for the time being.

**Comment by Nenormaln1** [comment]
Do you know if it's even possible to fix by now?

**Comment by pwall2222** [comment]
> Do you know if it's even possible to fix by now?

I am not sure yet, but I will close this issue if it can't be fixed.

**Comment by pwall2222** [comment]
Right now this issue will be stale for a while. Because RiotGames changed how their API works, we used to grab the `PartyID` from the local endpoint `/chat/v4/presences` but now that API only shows friends, which in part means that we can't fetch the `PartyID` for the players in our match. So we can't know who is premade.

TLDR; Can't look at parties because we can't get info from players

**Comment by pwall2222** [comment]
Update in the issue, I have seen some possibilities to show some kind of parties (not 100% accurate) but it will be better than not having any party indicators. It will take a while to implement.

Also for the technical details, the `tracker.gg` app uses last 20 matches to check if players have queued together and uses that to predict parties. Another way is thru an obscure endpoint feature that may or may not work.

**Comment by pwall2222** [comment]
I have a small update, the PartyID so the functionalty can be as good as before the API change can be retrieved via the messaging protocol of Riot (based off XMPP) so I am working in a side project to be able to bring that to NOWT. I don't have a estimated timeline so don't get too hyped but it'll be soon™.

**Comment by bossotron13** [comment]
update?

**Comment by pwall2222** [comment]
> update?

There is no update. For personal and general reasons, I haven't been involved a lot on finishing this. But as I stated previously don't get your hopes up it will be "soon™" refering that it could be from now till the end on time.

References: [Soon™](https://wowpedia.fandom.com/wiki/Soon)

**Comment by mfreewill** [comment]
it is not working anymore, not loading any info, even not showing that valorant is running 

Appreciate your efforts' dev,
